src/mlpack/tests/rl_environment_test.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file rl_environment_test.hpp
+ * @author Shangtong Zhang
+ *
+ * Basic test for the reinforcement learning task environment
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#include <mlpack/core.hpp>
+
+#include <mlpack/methods/reinforcement_learning/environment/mountain_car.hpp>
+#include <mlpack/methods/reinforcement_learning/environment/cart_pole.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace mlpack;
+using namespace mlpack::rl;
+
+BOOST_AUTO_TEST_SUITE(RLEnvironmentTest)
+
+BOOST_AUTO_TEST_CASE(MountainCarTest)
```

Comment:
I would call the test ``SimpleMountainCarTest`` since we might add some more tests here for the different environments.